.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,37 @@
+name: .NET
```

Comment:
Probably need to merge with master again since I feel this file shouldn't show up as diff?
https://github.com/Alying/team_ase_sharp/blob/main/.github/workflows/dotnet.yml